blobby/client.py

```diff
@@ -1,82 +0,0 @@
-from functools import partial
-from typing import Optional
-
-import boto3.session
-import botocore.exceptions
-from boto3.resources.base import ServiceResource
-
-from . import blob
-
-
-class StorageError(Exception):
-    pass
-
-
-class StorageClient:
-    chunk_size = 1024
-
-    def __init__(self,
-                 store_name: str,
-                 endpoint_url: str,
-                 s3: Optional[ServiceResource] = None) -> None:
-        self.session = boto3.session.Session()
-        if s3 is None:
-            self.s3 = self.session.resource(
-                's3', endpoint_url=endpoint_url
-            )  # pragma: no cover
-        else:
-            self.s3 = s3
-
-        self.bucket = self.s3.Bucket(store_name)
-
-    def get(self, key: str) -> bytes:
-        """Get the contents of an object.
-
-        Use get_reader to retrieve an object with more flexibility for
```

Comment:
I think this would be clearer without repetition of 'retrieve' - perhaps something like:
`Use get_reader where more flexibility is required for object retrieval` ?

Reply:
:+1: